src/molecule/config.py

```diff
@@ -221,6 +221,8 @@ def provisioner(self):
         provisioner_name = self.config["provisioner"]["name"]
         if provisioner_name == "ansible":
             return ansible.Ansible(self)
+        elif provisioner_name == "ansible-navigator":
+            return ansible_navigator.Ansible_Navigator(self)
```

Comment:
```suggestion
            return ansible_navigator.AnsibleNavigator(self)
```

Reply:
I applied these challenges in the last commit